docs/src/manual/variables.md

```diff
@@ -1308,6 +1308,40 @@ julia> x = @variable(model, [1:3], set = SecondOrderCone())
     You cannot delete the constraint associated with a variable constrained on
     creation.
 
+To check if a variable was constrained on creation, use [`has_variable_in_set`](@ref),
```

Comment:
```suggestion
To check if a variable was constrained on creation, use [`is_variable_in_set`](@ref),
```

Reply:
Is this a general request to change the name of the function? We have `is_integer` and `is_fixed` but `has_lower_bound` and `has_upper_bound`.

Reply:
A variable has a lower bound but you cannot say that a variable has a variable in set. I know you mean "has the variable an in set constraints" but that's not how it reads.

Reply:
Changed to `is`